### PR TITLE
Consolidate items on esintl tab (issue #1062)

### PR DESCRIPTION
**Pull request by afmenez**
This patch implements the suggestion on issue #1062.  I grouped similar issues together to simplify the page.

But I found one issue that may block this patch: #1067.

### CODE REVIEW

File: data-esintl.js [hunk 1]
```diff
@@ -150,48 +150,38 @@ exports.tests = [
...
-      res: {
-        ie11: true,
-        edge12: true,
-        firefox29: true,
-        chrome24: true,
-        safari10: true,
-        node012: true,
-      },
-    },
+	{
```
Comment (ljharb) [inline data-esintl.js]: i think you may want to `s/\t/  /g` in this patch?

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by afmenez** [comment]
I did the tab/space replace as requested by @ljharb , but I am not able to reply on the review.

**Review by chicoxyzzy — APPROVED** [review]

**Comment by ljharb** [comment]
I'm currently blocking based on https://github.com/kangax/compat-table/issues/1062#issuecomment-291633517

**Comment by afmenez** [comment]
Closing due to opposition by @ljharb .

**Pull request closed without merging**